realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -1827,6 +1827,7 @@ public void isValid_removedParent() {
         list.removeAll(SUPPORTED_IS_EMPTY_TYPES);
         list.remove(RealmFieldType.UNSUPPORTED_MIXED);
         list.remove(RealmFieldType.UNSUPPORTED_TABLE);
+        list.remove(RealmFieldType.UNSUPPORTED_DATE);
```

Comment:
Any comment to this? Maybe just have a list of *all* types inside `RealmFieldType` in a comment, but only expose the supported types?